### ISSUE TITLE
Error you exceeded your current quota

### ISSUE THREAD

**Issue by pcantalupo** [mckaywrigley/ai-brainstore#7]
Hello,

I installed chroma and started it with docker-compose.

```
$ docker container ls --all
CONTAINER ID   IMAGE                                      COMMAND                  CREATED          STATUS          PORTS                                                      NAMES
b8f0336a053d   chroma-server                              "uvicorn chromadb.ap…"   29 minutes ago   Up 29 minutes   0.0.0.0:8000->8000/tcp                                     chroma-server-1
f586204dae2a   clickhouse/clickhouse-server:22.9-alpine   "/entrypoint.sh"         29 minutes ago   Up 29 minutes   0.0.0.0:8123->8123/tcp, 0.0.0.0:9000->9000/tcp, 9009/tcp   chroma-clickhouse-1
```

Then ran `npm run agent`. After 1 minute, I got this error:
```
file:///Users/me/local/opt/ai-brainstore/node_modules/langchain/dist/util/axios-fetch-adapter.js:316
    const error = new Error(message);
                  ^
Error: Request failed with status code 429
    at createError (file:///Users/me/local/opt/ai-brainstore/node_modules/langchain/dist/util/axios-fetch-adapter.js:316:19)
    at settle (file:///Users/me/local/opt/ai-brainstore/node_modules/langchain/dist/util/axios-fetch-adapter.js:25:16)
    at file:///Users/me/local/opt/ai-brainstore/node_modules/langchain/dist/util/axios-fetch-adapter.js:175:19
    at new Promise (<anonymous>)
    at fetchAdapter (file:///Users/me/local/opt/ai-brainstore/node_modules/langchain/dist/util/axios-fetch-adapter.js:167:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async RetryOperation._fn (/Users/me/local/opt/ai-brainstore/node_modules/p-retry/index.js:50:12) {
```

that ended with :

```
    data: {
      error: {
        message: 'You exceeded your current quota, please check your plan and billing details.',
        type: 'insufficient_quota',
        param: null,
        code: null
      }
    }
```

What am I doing wrong? I'm currently using ChatGPT 3.5 on the free trial. Do I need to set up a paid account? 

Thank you

**Comment by mckaywrigley** [comment]
set up paid account

**Comment by pcantalupo** [comment]
I set up a paid account on ChatGPT for $20 per month.  I set the model to gpt-4 in my `.env` file.

`DEFAULT_MODEL=gpt-4`

Still, I'm getting the same error. I don't see another plan to choose from. Here is an image of my plan.  What is going wrong? Thank you
<img width="326" alt="Screen Shot 2023-04-30 at 7 32 24 AM" src="https://user-images.githubusercontent.com/310962/235350985-a05e3035-2096-407b-8e85-9f62eebd7a3e.png">
